srcpkgs/hashboot/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'hashboot'
 pkgname=hashboot
 version=0.9.14
-revision=1
+revision=2
```

Comment:
the revbump is not necessary. The builders will automatically pick up a changed template if the version hasn't been built for the architecture yet.